### ISSUE TITLE
Migrate most of the standard settings from ZK ingame to chobby

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Chobby#158]
Many 'standard' game settings are handled ingame. These settings should be moved to the lobby when chobby is detected. Standard settings are mostly things not handled by particular widgets. These include
 * Sound settings
 * icon distance (exists in lobby but overridden ingame).

**Comment by Licho1** [comment]
Dunno about this.. icon distance is VERY game specific and should not be handled in lobby, because you dont see effect instantly.

Settings should be mostly where you see their effects.

**Comment by GoogleFrog** [comment]
The way I see it, people expect settings to be in the settings menu in the lobby. The ingame configuration should be more for UI customization.

**Comment by gajop** [comment]
Agreed with @GoogleFrog . The fewer stuff exclusive to the ingame menu, the better. ZK's largely a MP game and you don't have time to configure everything while you play.

Generally I'd like to see both menus unified into a single one, but that might require some thoughtful design.

**Comment by Licho1** [comment]
Well consider this:
 * the icon distance is "ingame customization" you lack feedback 
 * the ingame menu is much more comprehensive
 * the ingame menu looks better
 * the ingame menu has search, bindings and other features

So ingame menu looks superior to me. 
Many games have this slight mismatch, for example in KSP in-menu settings differ from in-game settings.
I would only keep very high level gfx tings in chobby menu (like resolution and general preset), rest should stay "in  game".

**Comment by Licho1** [comment]
At best you can duplicate some things but you should not put any exclusives to chobby menu.

**Comment by Orfelius** [comment]
Having to run a game in order to set up it properly can be a big hassle. Having the in-game options avaible in Chobby would be great.

@Licho1 sure the in-game menu is superior but it isn't a reason why the options should not be avaible when the game is not running.

**Comment by Licho1** [comment]
Thats pretty much standard, you dont know what those things are in the lobby and lobby does not know what exists.
Apart from run of the mill graphics settings its all pretty much game specific.

**Comment by GoogleFrog** [comment]
"run of the mill" is "standard". We're lacking some sound settings as well as a few other things that I'll look through the ingame menu for.
